src/modules/displayData.js

```diff
@@ -0,0 +1,15 @@
+const display = document.querySelector('.names-scores');
+
+const displayData = (LeaderData) => {
+  if (!LeaderData.length) display.innerHTML = 'p class=\'details\'>No scores available!</p>';
+
+  LeaderData.forEach((data) => {
+    const nameUser = document.createElement('p');
+    nameUser.classList.add('details');
+    nameUser.id = 'details';
+    display.appendChild(nameUser);
+    nameUser.innerHTML = `${data.name} : ${data.score}`;
+  });
+};
```

Comment:
- [ ] Kindly make sure to implement a hard-coded (for now) list of scores to comply with the given wireframe design. Currently, there's a bug with how your elements are showing. (See attached below)
![image](https://user-images.githubusercontent.com/62290545/198056723-3068b19a-0932-4158-8770-524d097067dc.png)